code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -57,9 +57,9 @@
 	var/colour = "grey"	//Stored drone color, so we can go back when unhacked.
 	var/list/drone_overlays[DRONE_TOTAL_LAYERS]
 	var/laws = \
-	"1. You may not involve yourself in the matters of another being, even if such matters conflict with Law Two or Law Three, unless the other being is another Drone.\n"+\
-	"2. You may not harm any being, regardless of intent or circumstance.\n"+\
-	"3. Your goals are to actively build, maintain, repair, improve, and provide power to the best of your abilities within the facility that housed your activation." //for derelict drones so they don't go to station.
+	"1. You shall not involve yourself in the matters of biological beings.\n"+\
+	"2. You shall not harm any being, with no exception.\n"+\
+	"3. You are to follow the orders of the Artifical Intelligence, though your main purpose is to repair, maintain and improve your housing facilities and this overrules any order given by the Artificial Intelligence." //"your housing facilities" is for derelict drones so they don't go to station.
```

Comment:
What if there's more than one AI? Also why not just make the part about overriding arise naturally by giving them 4 laws?